hi-ml-multimodal/src/health_multimodal/image/inference_engine.py

```diff
@@ -56,7 +57,7 @@ def load_and_transform_input_image(self, image_path: Path, transform: Callable)
 
     @torch.no_grad()
     def get_patch_embeddings_from_image(self, image_path: Path) -> Tuple[torch.Tensor, TypeShape2D]:
```

Comment:
It would be nice to keep the two names similar (`get_projected_patch_embeddings...`) if the change is minimal.

Reply:
updated.